programs/account-compression/src/instructions/initialize_address_merkle_tree.rs

```diff
@@ -42,6 +46,45 @@ pub fn process_initialize_address_merkle_tree<'info>(
                 .map_err(|_| AccountCompressionErrorCode::IntegerOverflow)?,
         )
         .map_err(ProgramError::from)?;
+    let init_value = BigUint::from_str_radix(
+        "21888242871839275222246405745257275088548364400416034343698204186575808495617",
```

Comment:
To be precise - is this what a constant in `ark_bn254::Fr::MODULUS` would be? I would prefer to use it that constant instead. Putting it like this is quite cryptic and if I never worked with prime fields, I would have no idea what it stands for.

Reply:
Introducing a dependency for just one constant is too much in my opinion. Happy to make it a constant.

Reply:
Fair enough. But that said, ark-ff and and ark-bn254 are already dependencies of account-compression

```
[🧢 Light Protocol devenv] node@e5c508d57ec7:~/light-protocol/programs/account-compression$ cargo +solana tree -p account-compression | grep ark-
warning: some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
│   ├── ark-ff v0.4.2
│   │   ├── ark-ff-asm v0.4.2 (proc-macro)
│   │   ├── ark-ff-macros v0.4.2 (proc-macro)
│   │   ├── ark-serialize v0.4.2
│   │   │   ├── ark-serialize-derive v0.4.2 (proc-macro)
│   │   │   ├── ark-std v0.4.0
│   │   ├── ark-std v0.4.0 (*)
│   │   │   │   ├── ark-bn254 v0.4.0
│   │   │   │   │   ├── ark-ec v0.4.2
│   │   │   │   │   │   ├── ark-ff v0.4.2 (*)
│   │   │   │   │   │   ├── ark-poly v0.4.2
│   │   │   │   │   │   │   ├── ark-ff v0.4.2 (*)
│   │   │   │   │   │   │   ├── ark-serialize v0.4.2 (*)
│   │   │   │   │   │   │   ├── ark-std v0.4.0 (*)
│   │   │   │   │   │   ├── ark-serialize v0.4.2 (*)
│   │   │   │   │   │   ├── ark-std v0.4.0 (*)
│   │   │   │   │   ├── ark-ff v0.4.2 (*)
│   │   │   │   │   └── ark-std v0.4.0 (*)
│   │   │   │   ├── ark-ec v0.4.2 (*)
│   │   │   │   ├── ark-ff v0.4.2 (*)
│   │   │   │   ├── ark-serialize v0.4.2 (*)
│   │   │   │   │   ├── ark-bn254 v0.4.0 (*)
│   │   │   │   │   ├── ark-ff v0.4.2 (*)
│   │   │   ├── ark-bn254 v0.4.0 (*)
│   │   │   │   ├── ark-bn254 v0.4.0 (*)
│   │   │   │   ├── ark-ff v0.4.2 (*)
│       ├── ark-ff v0.4.2 (*)
│       │   ├── ark-bn254 v0.4.0 (*)
│       │   ├── ark-ff v0.4.2 (*)
├── ark-ff v0.4.2 (*)
├── ark-serialize v0.4.2 (*)
│   ├── ark-ff v0.4.2 (*)
```

If you want to kick them out, it would be great to do in a separate PR

Reply:
fair point, we should take a look what we can do about this.
We probably also have arkworks crates in groth16 solana.